content/talks/_index.md

```diff
@@ -8,28 +8,19 @@ weight: 100
 ## Papers
 
 * SC 21, [Reverse-Mode Automatic Differentiation and Optimization of GPU Kernels via Enzyme](https://dl.acm.org/doi/abs/10.1145/3458817.3476165); William S. Moses, Valentin Churavy, Ludger Paehler, Jan Hückelheim, Sri Hari Krishna Narayanan, Michel Schanen, and Johannes Doerfert
```

Comment:
Can we update these?

at minimum we can pull the relevant ones from https://wsmoses.com/academic/ and enzyme.mit.edu/conference


We should also link to enzyme.mit.edu/conference